app/views/organizations/new.html.erb

```diff
@@ -16,7 +16,7 @@
       <div class="d-flex flex-wrap gutter-spacious">
         <% @users_github_organizations.each do |org| %>
           <div class="col-6 col-md-4 col-lg-3 mb-6">
-            <% if org[:role] == 'admin' && (!org[:classroom].present? || multiple_classrooms_per_org_enabled?) %>
```

Comment:
Does removing `!org[:classroom].present?` do anything? Just curious on what that line was for.

Reply:
I did not notice any difference in behavior after removing it, but maybe @srinjoym can help us out here